source/reference/minio-mc/mc-sql.rst

```diff
@@ -106,71 +102,62 @@ Parameters
    - ``SNAPPY`` `Snappy <http://google.github.io/snappy/>`__ framed stream
 
 .. mc-cmd:: --csv-output
-   
+   :optional:
 
-   *Optional* The data format for ``.csv`` output. Specify a string of
-   comma-seperated ``key=value,...`` pairs. See :ref:`mc-sql-csv-format` for
-   more information on valid keys.
+   The data format for ``.csv`` output.
+   Specify a string of comma-seperated ``key=value,...`` pairs.
+   See :ref:`mc-sql-csv-format` for more information on valid keys.
 
-   See the S3 API :s3-api:`CSVOutput <API_CSVOutput.html>` for more
-   information.
+   See the S3 API :s3-api:`CSVOutput <API_CSVOutput.html>` for more information.
 
 .. mc-cmd:: --csv-output-header
-   
+   :optional:
 
-   *Optional* The header row of the ``.csv`` output file. Specify a string of
-   comma-separated fields as ``field1,field2,...``.
+   The header row of the ``.csv`` output file.
+   Specify a string of comma-separated fields as ``field1,field2,...``.
 
    Omit to output a ``.csv`` with no header row.
 
 .. mc-cmd:: --encrypt-key
-   
+   :optional:
 
-   *Optional* The encryption key to use for performing Server-Side Encryption
-   with Client Keys (SSE-C). Specify comma seperated key-value pairs as
-   ``KEY=VALUE,...``.
-   
-   - For ``KEY``, specify the MinIO deployment 
-     :mc-cmd:`alias <mc alias>` and full path to the bucket, including any
-     bucket prefixes. Separate the alias and bucket path with a forward slash 
-     ``\``. For example, ``play/mybucket``
+   The encryption key to use for performing Server-Side Encryption with Client Keys (SSE-C).
+   Specify comma seperated key-value pairs as ``KEY=VALUE,...``.
 
-   - For ``VALUE``, specify the data key to use for encryption object(s) in
-     the bucket or bucket prefix specified to ``KEY``.
+   - For ``KEY``, specify the MinIO deployment :mc-cmd:`alias <mc alias>` and full path to the bucket, including any bucket prefixes.
+     Separate the alias and bucket path with a forward slash ``\``.
+     For example, ``play/mybucket``.
 
-   :mc-cmd:`~mc sql --encrypt-key` can use the ``MC_ENCRYPT_KEY``
-   environment variable for populating the list of encryption key-value
-   pairs as an alternative to specifying them on the command line.
+   - For ``VALUE``, specify the data key to use for encryption object(s) in the bucket or bucket prefix specified to ``KEY``.
+
+   :mc-cmd:`~mc sql --encrypt-key` can use the ``MC_ENCRYPT_KEY`` environment variable for populating the list of encryption key-value pairs as an alternative to specifying them on the command line.
```

Comment:
```suggestion
   :mc-cmd:`~mc sql --encrypt-key` can use the :envvar:`MC_ENCRYPT_KEY` environment variable for populating the list of encryption key-value pairs as an alternative to specifying them on the command line.
```

Reply:
Ooo there's one in `mc cp` also. Might as well. 👍🏻

Reply:
Why why why is editing something through the GitHub UI so annoying? Fixing the things the normal way.